gradlew.bat

```diff
@@ -1,104 +1,104 @@
-@rem

```

Comment:
I cannot get git to leave this file alone. It changes it to have CRLF but according to the .gitattributes that should already be the case.

Reply:
yeah, we just changed the `.gitattributes` file.

can you try:

```
rm .gitattributes
git checkout -- .gitattributes
```

Reply:
oh i just got stuck in this again now too, will debug, really not understanding this either

Reply:
this seems to make it go away at least temporarily:

```
sed -i '/.bat/d' .gitattributes && git status > /dev/null && git checkout -- .gitattributes
```

Reply:
i think i found permanent fix #743, sorry for the confusion 😅

Reply:
👍